interfaces/user-timing.idl

```diff
@@ -0,0 +1,17 @@
+
+        partial interface Performance {

```

Comment:
Can you update to strip the indentation and use the file in https://github.com/w3c/web-platform-tests/blob/master/user-timing/idlharness.html?

Reply:
@lukebjerring, indentation problem still persists.